js/app.js

```diff
@@ -0,0 +1,131 @@
+  },
+  {
+    sImage: ['./images/person1.jpg'],
+    name: 'Engr. John Samuel',
+    discipline: `A professor of the department of Civil Enineering in the
+    Unviversity og Lagos, Nigeria.`,
+    biography: 'He was born in the year 1975 and he is married with three kids',
+  },
+  {
+    sImage: ['./images/person2.jpg'],
+    name: 'Prof. Magret James',
+    discipline: `A professor of the department of Structural Enineering in the
+    Unviversity og Abuja, Nigeria.`,
+    biography:
+      'He was born in the United States of America in the year 1960 and she is married with two kids',
+  },
+  {
+    sImage: ['./images/person1.jpg'],
+    name: 'Engr. John Samuel',
+    discipline: `A professor of the department of Civil Enineering in the
+    Unviversity og Lagos, Nigeria.`,
+    biography: 'He was born in the year 1975 and he is married with three kids',
+  },
+  {
+    sImage: ['./images/person2.jpg'],
+    name: 'Prof. Magret James',
+    discipline: `A professor of the department of Structural Enineering in the
+    Unviversity og Abuja, Nigeria.`,
+    biography:
+      'He was born in the United States of America in the year 1960 and she is married with two kids',
+  },
+  {
+    sImage: ['./images/person1.jpg'],
+    name: 'Engr. John Samuel',
+    discipline: `A professor of the department of Civil Enineering in the
+    Unviversity og Lagos, Nigeria.`,
+    biography: 'He was born in the year 1975 and he is married with three kids',
+  },
+  {
+    sImage: ['./images/person2.jpg'],
+    name: 'Prof. Magret James',
+    discipline: `A professor of the department of Structural Enineering in the
+    Unviversity og Abuja, Nigeria.`,
+    biography:
+      'He was born in the United States of America in the year 1960 and she is married with two kids',
+  },
+];
+
+let speakerImage = '';
+let speakerName = '';
+let speakerDiscipline = '';
+let speakerBiography = '';
+
+speakerLists.forEach((n, index) => {
+  speakerImage = n.sImage;
+  speakerName = n.name;
+  speakerDiscipline = n.discipline;
+  speakerBiography = n.biography;
+
+  const content = `<div class="speaker_image">
+      <img class="check" src="./images/check_box.png" alt="First Speaker"/>
+      <img class="speak_image" src="${speakerImage}" alt="First Speaker" />
+    </div>
+    <div class="speaker_details">
+      <h4 class="speaker_name">${speakerName}</h4>
+      <p class="speaker_discipline">${speakerDiscipline}</p>
+      <p class="speaker_biography">${speakerBiography}</p>
+    </div>`;
+
+  const li = document.createElement('li');
+  li.classList.add('speaker_lists');
+  if (index > 1) {
+    li.classList.add('des_speaker_lists');
+  }
+  li.innerHTML = content;
+  ul.append(li);
+});
+const moreDiv = document.createElement('div');
+moreDiv.classList.add('more');
+const more = `
+<span class="more_speaker">More</span
+><img src="./images/down_arrow.png" alt="down arrow" />
+`;
```

Comment:
- [ ] Please add javascript functionalities to the more button that is visible for the mobile screens, When users click on the more button the remaining speakers should load on the screen.